runtime/ms-rest-azure/lib/azureEnvironment.js

```diff
@@ -124,6 +124,25 @@ var supportedEnvironments = {
     // TODO: add dns suffixes for the US government for datalake store and datalake analytics once they are defined.
     azureDataLakeStoreFileSystemEndpointSuffix: 'N/A',
     azureDataLakeAnalyticsCatalogAndJobEndpointSuffix: 'N/A'
+  },
+  AzureStack: {
+    name: 'AzureStack',
+    portalUrl: 'http://portal.microsoftazure.de/',
```

Comment:
How can the portal url be that of German cloud? This seems incorrect to me.

Reply:
oops... apparently it is not used in test